src/app/page/Signup/SignupScene.jsx

```diff
@@ -38,6 +21,41 @@ const convertFullName = (fullName) => {
 function SignupScene(props) {
   const { handleChange, values } = useForm();
   const [activeTab, setActiveTab] = useState(0);
+  const [errorSnackbarMessage, setErrorSnackbarMessage] = useState(false);
+  const [loading, setLoading] = useState(false);
+
+  function getPayload() {
+    return {
+      description: values.description || "",
+      displayName: `${values.firstName} ${values.lastName}`,
+      email: values.email || "",
+      isOrganizer: false,
+      isVolunteer: true,
+      location: values.location && convertLocation(values.location),
+      organizerDetails: {},
+      phone: values.phone || "",
+      photoUrl: "",
+      volunteerDetails: {
+        availability: values.availability || "",
+        hasTransportation: Boolean(values.hasTransportation),
```

Comment:
hasTransportation is already boolean isnt it?

Reply:
it could be undefined